src/python/pants/backend/go/module.py

```diff
@@ -44,10 +45,22 @@ class ModuleDescriptor:
 
 @dataclass(frozen=True)
 class ResolvedGoModule:
+    # Address of the resolved go_module target.
     address: Address
+
+    # The go_module target.
+    target: Target
+
+    # Import path of the Go module. Inferred from the import path in the go.mod file.
```

Comment:
Thanks for adding! Probably even better is to add formal docstring with `"""`. Then it shows up in IDE autocomplete etc. I think you'd use something like `:param: import_path: Minimum Go..."` for example.

Not a blocker.

(Ack that we are often using inline comments elsewhere, especially for class properties.)

Reply:
Added TODO.